.github/workflows/clone-firestore-indexes.yml

```
@@ -0,0 +1,72 @@
+name: Clone Firestore Indexes
+
+on:
+    workflow_call:
+        inputs:
+            original-project-id:
+                description: "The Firebase Project ID whose indexes are to be cloned"
+                required: true
+                type: string
+
+            target-project-id:
+                description: "The Firebase Project ID to which indexes are to be cloned"
```

```suggestion
                description: "The Firebase Project ID for receiving copied indexes"
```

might be a bit clearer